CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.1.0]
+### Changed
+- Confirm a transaction conditionally ([#301](https://github.com/MetaMask/smart-transactions-controller/pull/301))
```

Comment:
When does a transaction now get confirmed or not confirmed? Based on the commit message, perhaps this would be more informative?

Also, this PR seems to introduce a breaking change by requiring that the constructor is given a `getTransactions` option.

What do you think about this?

``` suggestion
- **BREAKING:** The constructor now requires a `getTransactions` option, which can be used to get a list of existing transactions ([#301](https://github.com/MetaMask/smart-transactions-controller/pull/301))
- Ensure that a transaction does not get re-confirmed it is already confirmed or submitted ([#301](https://github.com/MetaMask/smart-transactions-controller/pull/301))
```

Reply:
Basically, if the TransactionController is going to confirm it, we don't need to do it here. For MetaMask's Smart Swaps it always needs to be confirmed right now, but for other transactions it's done in the TransactionController almost every time, unless there was some issue (e.g. a user closed and re-opened their browser right after submitting a transaction).